services/database/package.json

```diff
@@ -9,19 +9,9 @@
   "author": "",
   "license": "CC0-1.0",
   "devDependencies": {
-    "@types/aws-lambda": "^8.10.108",
-    "aws-lambda": "^1.0.7",
     "aws-dynamodb-local": "^0.0.10",
-    "dotenv": "^16.0.3",
     "dynamodb-localhost": "https://github.com/99x/dynamodb-localhost#db30898f8c40932c7177be7b2f1a81360d12876d",
-    "serverless": "^3.27.0",
     "serverless-dynamodb": "^0.2.42",
-    "serverless-offline": "^12.0.4",
-    "serverless-plugin-scripts": "^1.0.2",
-    "serverless-plugin-typescript": "^2.1.4",
-    "serverless-s3-bucket-helper": "github:Enterprise-CMCS/serverless-s3-bucket-helper#master",
-    "serverless-s3-local": "^0.7.1",
-    "serverless-stack-termination-protection": "^2.0.2",
```

Comment:
these are either unused in database/serverless.yml or provided by the root package.json